.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this needed

Reply:
This is a fork by @mpasternak that has only one version for now. It's a nice alternative until the autoslug package is fixed.